### DISCUSSION TITLE
[Request] Dark Mode

### DISCUSSION THREAD

**Discussion by ayakpompasi** [windirstat/windirstat#142] [Q&A]
I updated to the latest version available. I love this program and don't want to replace it with another one. But it doesn't have a dark mode feature. Can you add it soon?

**Comment by NoMoreFood** [comment]
I think we'll need to define a minimum variable product of what constitutes "dark mode".  MFC (the ancient, efficient, and backward-compatible) display framework we are using is not as easily adapted to do dark modes as some newer frameworks.  It's easy enough to change the background color of the tree controls, the text controls, the selection color, etc... but the things like title bar, scrolls bars, and property dialogs can't be easily done.

**Comment by NoMoreFood** [comment]
Will be available in the next release.

**Comment by NoMoreFood** [comment]
Going to close this out since this is going to be in the next release.
